.github/actions/release-image/action.yml

```diff
@@ -48,16 +45,20 @@ runs:
     - shell: bash
       if: inputs.gcsFetchBucketName != ''
       run: |
-        set -x
-        mkdir -p "${{ github.workspace }}/packages"
-        mkdir -p /tmp/gcsfuse
+        mkdir -p "${{ github.workspace }}/packages/os"
+        mkdir -p "${{ github.workspace }}/packages/os/aarch64"
+        mkdir -p "${{ github.workspace }}/packages/os/x86_64"
+
+        # We only need the key, the index, and chainguard-baselayout
+        gsutil cp gs://"${{ inputs.gcsFetchBucketName }}"/os/chainguard-enterprise.rsa.pub "${{ github.workspace }}/packages/os"
+        
+        # Index (one for each arch)
+        gsutil cp gs://"${{ inputs.gcsFetchBucketName }}"/os/aarch64/APKINDEX.tar.gz "${{ github.workspace }}/packages/os/aarch64"
+        gsutil cp gs://"${{ inputs.gcsFetchBucketName }}"/os/x86_64/APKINDEX.tar.gz "${{ github.workspace }}/packages/os/x86_64"
 
-        # Install gcsfuse and mount the $BUCKET/os/ to ./packages/ via symlink
-        go install github.com/googlecloudplatform/gcsfuse@latest
-        gcsfuse "${{ inputs.gcsFetchBucketName }}" /tmp/gcsfuse
-        ln -s /tmp/gcsfuse/os/ "${{ github.workspace }}/packages/"
-        ls -al /tmp/gcsfuse/os/
-        ls -al "${{ github.workspace }}/packages/"
+        # Packages (one for each arch)
+        gsutil cp gs://"${{ inputs.gcsFetchBucketName }}"/os/aarch64/chainguard-baselayout-* "${{ github.workspace }}/packages/os/aarch64"
+        gsutil cp gs://"${{ inputs.gcsFetchBucketName }}"/os/x86_64/chainguard-baselayout-* "${{ github.workspace }}/packages/os/x86_64"
```

Comment:
what about all the other packages?

Reply:
We think this is the only one we need from the enterprise packages.

Reply:
Plan is to make it a config option.  That way the private images still get the full private repo.

Reply:
FYI this whole step only runs `if: inputs.gcsFetchBucketName != ''`, which shouldnt be the case in this repo